From da33289073de254ab4bacb80b1b83cf9d27c76ea Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Tue, 24 Aug 2021 14:10:20 +0200
Subject: [PATCH] tests: update expect files for f_large_dir and
 f_large_dir_csum

Update expect files for f_large_dir and f_large_dir_csum tests to
include the warning about missing y2038 support with 128-byte inodes.

Fixes: a23b50cd ("mke2fs: warn about missing y2038 support when formatting fresh ext4 fs")
Signed-off-by: Lukas Czerner <lczerner@redhat.com>
Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---
 tests/f_large_dir/expect      | 1 +
 tests/f_large_dir_csum/expect | 1 +
 2 files changed, 2 insertions(+)

diff --git a/tests/f_large_dir/expect b/tests/f_large_dir/expect
index 028234c..495ea85 100644
--- a/tests/f_large_dir/expect
+++ b/tests/f_large_dir/expect
@@ -1,3 +1,4 @@
+128-byte inodes cannot handle dates beyond 2038 and are deprecated
 Creating filesystem with 108341 1k blocks and 65072 inodes
 Superblock backups stored on blocks: 
 	8193, 24577, 40961, 57345, 73729
diff --git a/tests/f_large_dir_csum/expect b/tests/f_large_dir_csum/expect
index aa9f33f..44770f7 100644
--- a/tests/f_large_dir_csum/expect
+++ b/tests/f_large_dir_csum/expect
@@ -1,3 +1,4 @@
+128-byte inodes cannot handle dates beyond 2038 and are deprecated
 Creating filesystem with 31002 1k blocks and 64 inodes
 Superblock backups stored on blocks: 
 	8193, 24577
-- 
1.8.3.1

